end2end/tests/reportBuilder.spec.ts

```diff
@@ -2,30 +2,208 @@ import { test, expect } from '@playwright/test';
+    await page.getByRole('button', { name: 'Modify Search' }).click();
+    await page.getByLabel('add logical and filter').click();
+    await page.getByRole('cell', { name: 'Resolved' }).locator('a').click();
+    await page.getByRole('option', { name: 'Submitted', exact: true }).click();
+    await page.getByRole('button', { name: 'Next Step' }).click();
+    await page.locator('#indicatorList').getByText('General Form').click();
+    await page.getByTitle('indicatorID: 7\nRadio').locator('span').click();
+    await page.getByRole('button', { name: 'Generate Report' }).click();
+    await expect(page.getByLabel('Sort by Radio')).toBeVisible();
+});
+
+test("fill textbox for multi line text", async ({ page }) => {
+    await page.goto("https://host.docker.internal/Test_Request_Portal/?a=reports&v=3&query=N4IgLgpgTgtgziAXAbVASwCZJHSAHASQBEQAaEAez2gEMwKpsBeMkGOgYwAtsoI4KAGwBuELOQDmdCNgCCAORIBfUuiyIQHaRIYBPYqyq16jDS3Lsw3bADMGMAPoBWCDQAMAdlZTIcxSBU1bAwIQQhIcUpqKDoGZlZLa0Q3SWk%2FZQBdcgArCjQAOwQUEAK0MDRYqHkaGBksnAYwJGAVEAlwojoaJGQQABZWL3IAZhB6wTQYMqQARjd58gmpsAB5Gxs4cKQ3JSA%3D%3D&indicators=NobwRAlgdgJhDGBDALgewE4EkAiYBcYyEyANgKZgA0YUiAthQVWAM4bL4AMAvpeNHCRosuAtGIQUGZrQb4wzNug54efSLARSR8xPCKooAIQCuyNFBn1GC6kpVr%2BmoRhzyALFbkEAsiZJEAAQk0GSByGQAHhx27Fy8ToLabgQA7F42AEqIcKiKcaoJGknCKWAAzBnyAMrQAObkwaHhUTGsBTwAukA");
+    await page.getByRole("cell", { name: "test", exact: true }).click();
+    await page.getByLabel("Multi line text", { exact: true }).click();
+    await page.getByLabel('Multi line text', { exact: true }).fill("test this is a test");
+    await page.getByRole("button", { name: "Save Change" }).click();
+    await expect(page.getByRole('cell', { name: "test this is a test" })).toBeVisible();
+
+    await page.getByRole("cell", { name: "test this is a test" }).click();
+    await page.getByLabel("Multi line text", { exact: true }).click();
+    await page.getByLabel('Multi line text', { exact: true }).fill('test');
+    await page.getByRole('button', { name: 'Save Change' }).click();
+});
+
+test("Select multiple filter using AND/OR,", async ({ page }) => {
+    await page.goto("https://host.docker.internal/Test_Request_Portal/?a=reports&v=3");
+    await page.getByRole('cell', { name: 'IS NOT' }).locator('a').click();
+    await page.getByRole('option', { name: 'IS', exact: true }).click();
+    await page.getByRole('cell', { name: 'Current Status' }).locator('a').click();
+    await page.getByRole('option', { name: 'Type', exact: true }).click();
+    await page.getByRole('cell', { name: 'Complex Form' }).locator('a').click();
+    await page.getByRole('option', { name: 'General Form' }).click();
+    await page.getByLabel('add logical and filter').click();
+    await page.getByRole('button', { name: 'Next Step' }).click();
+    await page.getByRole('button', { name: 'Generate Report' }).click();
+});
+
+test("Edit Report Title", async ({ page }) => {
```

Comment:
```
Error: locator.click: Test timeout of 30000ms exceeded.
Call log:
  - waiting for getByRole('cell', { name: 'TestForm_NonadminCannotCancelOwnSubmittedRecord_new' })
```

Your database probably includes content that doesn't exist in the standardized version. Run the API tester to restore your local database back to the standard.

Reply:
Fixed